src/CMakeLists.txt

```diff
@@ -35,6 +38,15 @@ protobuf_gen(PROTO ${kuksa_proto}
     OUTPUT ${proto_gen_dir}
 )
 include_directories("${proto_gen_dir}")
+add_library(${GRPC_GENERATED_LIB_NAME} OBJECT ${PROTO_SRCS}
```

Comment:
add generated grpc code as separate lib, since there is a cmake problem for collecting coverage information. The codecov cmake always use a wrong path for the gerade path under build directory. I think it is a bug of the codecov cmake module, but I can not figure out, how to fix it. 
But I think, it is not necessary to collect coverage information for generated code. Therefore, I define it as a separate lib to workaround the problem.